src/Compilers/CSharp/Portable/Binder/Semantics/OverloadResolution/OverloadResolution.cs

```diff
@@ -1871,12 +1860,23 @@ private BetterResult BetterConversionFromExpression(BoundExpression node, TypeSy
                         return conv2.Kind == ConversionKind.Identity ? BetterResult.Right : BetterResult.Neither;
                     }
                 }
-                else if (refKind1 == RefKind.Ref)
+                else if (p1.RefKind == RefKind.Ref)
                 {
                     return BetterResult.Neither;
                 }
             }
 
+            // If an ambiguity exists between 'by-val' and 'in' parameter, choose the 'by-val' one, except if it was params,
+            // for later betterness analysis will decide on whether the expanded/non-expanded form can be used, or if it is ambiguous.
+            if (!p1.IsParams && p1.RefKind == RefKind.None && p2.RefKind == RefKind.In)
```

Comment:
>p1.IsParams [](start = 17, length = 11)

See IsValidParams helper how to check for valid params parameter. Also, why should this matter if candidate is not using expanded form?